dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10-slim-bullseye
+
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN pip --no-cache-dir install --ignore-installed distlib -r requirements.txt
+RUN pip install gunicorn
+COPY . .
+
+EXPOSE 8080
+
+CMD ["gunicorn", "-w", "1", "-b", "0.0.0.0:8080", "app:create_app()"]
```

Comment:
This looks fine but in general I would rather unrelated things go into separate PRs rather than bundling in stuff which is not in the story.

Reply:
Good work but I agree, separate PR's causes less confusion.

Reply:
Yep was for local work shouldn't have committed

Reply:
Removed